hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieAppendHandle.java

```diff
@@ -153,7 +154,7 @@ public HoodieAppendHandle(HoodieWriteConfig config, String instantTime, HoodieTa
             // When enabling writing partial updates to the data blocks in log files,
             // i.e., partial update schema is set, the writer schema is the partial
             // schema containing the updated fields only
-            ? Option.of(new Schema.Parser().parse(config.getPartialUpdateSchema()))
+            ? Option.of(AvroSchemaUtils.parseSchemaFromStr(config.getPartialUpdateSchema()))
```

Comment:
Move the method to `AvroSchemaCache` and rename it to just `AvroSchemaCache#getAvroSchema`

Reply:
> Move the method to `AvroSchemaCache` and rename it to just `AvroSchemaCache#getAvroSchema`

I removed it.